.github/workflows/ci-spell-checking.yml

```diff
@@ -0,0 +1,44 @@
+name: ci-spell-checking
+
+on:
+  push:
+    paths-ignore:
+      - '.vscode/styles/**'
+    branches: 
+    - main
+  pull_request:
+    types:
+    - opened
+    - reopened
+    - synchronize
+    - ready_for_review
+    branches: 
+    - main
+
+jobs:
+  spell-check:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+      - name: Get changed files in the content folder
+        id: changed-files-specific
+        uses: tj-actions/changed-files@v35
+        with:
+          files: ./
+          files_ignore: './.vscode/'
+      - name: Install Vale
+        if: steps.changed-files-specific.outputs.any_changed == 'true'
+        run: |
+          sudo apt-get update
+          sudo apt-get install curl
+          curl -Ls https://api.github.com/repos/errata-ai/vale/releases/latest \
+            | grep "browser_download_url.*vale_[0-9.]*_Linux_64-bit.tar.gz" \
+            | cut -d : -f 2,3 \
+            | tr -d \" \
+            | xargs curl -Ls \
+            | tar xz
+            sudo mv ./vale /usr/bin/vale

```

Comment:
we don't need to do this. We can run the command from a local folder.

Reply:
I did this before, I ran into some issues about setting up Vale. So doing it that way solved the problems I had at the time. I don't remember anymore, but I remember that I did it to solve a specific problem in the Vale.